content/guides/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: IPFS Guides
+headless: true
+opened: true
+---
+
+# [Install IPFS](https://docs.ipfs.io/guides/install/)
+
+Install page describes the variety of ways you can have IPFS on the machine.
```

Comment:
_@tapaswenipathak, I suggest changing this section to the following for clarity:_

Instructions for installing a copy of IPFS on your system via your choice of the following options:

- Installing from a Prebuilt Package (recommended)
- Installing from ipfs-update
- Building from source

Also includes information on how to upgrade IPFS, and general troubleshooting instructions.